File: backend-greengarden/GreenGardenAPI/Models/Garden.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json;
using System.Text.Json.Serialization;

namespace GreenGardenAPI.Models
{
    public class Garden
    {
        public int? Id { get; set; }
        public string? Name { get; set; }
        public string? Description { get; set; }
        public string? Phone { get; set; }
        public string? Objective { get; set; }
        public string? Adress { get; set; }
        public string? BannerURL { get; set; }
        public string? ImageURL { get; set; }
        public string? LogoURL { get; set; }
        public List<Products>? Products { get; set; }

        [ForeignKey("User")]
        public int UserId { get; set; }
        public virtual User? User { get; set; }
    }
}

File: frontend-greengarden/garden.js
function getGardenName() {

  const urlParams = new URLSearchParams(window.location.search);
  const gardenName = urlParams.get("gardenName") // livros

  return gardenName;
}

function getGardenData(name) {
  // let name = document.getElementById('name').value;
  const gardemName = localStorage.gardenName;
   console.log(gardemName)

  fetch('https://localhost:7002/Garden/'.concat(localStorage.userId).concat('/v2'))
    .then(function (serverPromise) {
      serverPromise.json()
        .then(function (result) {
          console.log('result', result);
          document.getElementById('gardenName').innerHTML = `${result.name}`;
          document.getElementById('gardenPageTitle').innerHTML = `${result.name}`;
          document.getElementById('gardenNameMobile').innerHTML = `${result.name}`;
          document.getElementById('gardenPhone').innerHTML = `${result.phone}`;
          document.getElementById('footerPhone').innerHTML = `${result.phone}`;
          document.getElementById('gardenDescription').innerHTML = `${result.description}`;
          document.getElementById('objective').innerHTML = `${result.objective}`;
          document.getElementById('adress').innerHTML = `${result.adress}`;
          // document.getElementById('banner').innerHTML = `${result.bannerURL}`.outerHTML;
          document.getElementById("banner").src=`${result.bannerURL}`;
          document.getElementById("imageCenter").src=`${result.imageURL}`;
          document.getElementById("logoImage").src=`${result.logoURL}`;


        })
        .catch(function (error) {
          console.log(error);
        });
    })
    .catch(function (error) {
      console.log(error);
    });

}

// ao carregar a pagina GARDEN
(function () {

  getGardenData();


})();